dashboard/app/controllers/api/v1/pd/workshops_controller.rb

```diff
@@ -116,6 +116,12 @@ def show
   # PATCH /api/v1/pd/workshops/1
   def update
     adjust_facilitators
+
+    # Add regional partner permission here when those are added
```

Comment:
I don't understand this comment. Is this meant to be a TODO?

Reply:
It's a note to Clare that her work needs to go here as well.

Reply:
So please prefix it with a `TODO` (or more specifically `TODO (Clare):`. The way I read this comment is that it's describing the following code, as in the next few lines add a regional partner permission, but then the "when those are added" part is confusing, and also it doesn't describe the following lines.